infrastructure-playbooks/rolling_update.yml

```diff
@@ -194,6 +194,19 @@
                   when: (check_quorum_status.stdout | from_json).quorum | length != groups[mon_group_name] | length
               when: inventory_hostname == groups[mon_group_name] | first
 
+        - name: Replace systemd docker files with updates
+          when: container_binary == 'docker'
+          block:
+            - import_role:
+                name: ceph-mon
+                tasks_from: systemd.yml
+
+            - name: reload docker service
+              systemd:
+                name: docker
+                state: reloaded
+                daemon_reload: true
```

Comment:
```suggestion
            - name: reload docker service
              systemd:
                name: docker
                state: reloaded
                daemon_reload: true
```

The service you have to reload isn't docker itself, it's `ceph-mon@{{ ansible_facts['hostname'] }}`